site/en/docs/web-platform/origin-trial-troubleshooting/index.md

```diff
@@ -167,13 +167,17 @@ For examples of pages that _do_ include an origin trial token, see the [demos](#
 
 ## Chrome DevTools status codes {: #devtools-status}
 
-* **Success**: The token is well-formed, has not expired, matches an origin trial feature, and is
-requested from an expected origin.<br>
-[Source code](https://source.chromium.org/chromium/chromium/src/+/main:third_party/blink/common/origin_trials/trial_token.cc;l=84)
+* **Expired**: Token has passed its expiration date. The token will need to be renewed, to
+generate a new token with a new expiration date.<br>
```

Comment:
```suggestion
* **Expired**: The token has passed the expiration date. Tokens cannot be automatically
updated, so developers must renew participation in the origin trial to generate a new
token (with a new expiration date).<br>
```

Reply:
A weird thing about this process is the action "Renew" being tied to participation in the OT, rather than the token itself. "Renew" generates a new token. So I've tried to extrapolate that a bit in my suggestion.

Reply:
Yeah... I agree. Good call.